src/lib/bakeryjs/ComponentFactory.ts

```diff
@@ -77,13 +77,18 @@ export class ComponentFactory implements ComponentFactoryI {
 		componentsPath: string,
 		parentDir: string = ''
 	): void {
-		const files = fs.readdirSync(componentsPath);
+		const componentsPathSanit = componentsPath.endsWith('/')
+			? componentsPath
+			: `${componentsPath}/`;
+		const files = fs.readdirSync(componentsPathSanit);
 		files.forEach(
 			(file: string): void => {
-				if (fs.statSync(`${componentsPath}${file}`).isDirectory()) {
+				if (
+					fs.statSync(`${componentsPathSanit}${file}`).isDirectory()
+				) {
 					if (file !== '.' && file !== '..') {
 						this.findComponents(
-							`${componentsPath}${file}/`,
+							`${componentsPathSanit}${file}/`,
```

Comment:
Why not just use `path.join` and avoid this string juggling?

```suggestion
							path.join(componentsPath, file),
```